### DISCUSSION TITLE
Increase the response area of buttons

### DISCUSSION THREAD

**Discussion by FaqatZubair1** [libre-tube/LibreTube#5709] [General]
### Describe your suggested feature

Whenever I try clicking on queue button to view the queue, it doesn't click in the first try. The response area of the buttons is very small. Increase the response area so that a person with big fingers can also navigate around easily.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
I've been trying to intentionally miss the button by a bit and it still triggered the queue, I don't think the area is too small as of now.

**Comment by FaqatZubair1** [comment]
[

https://github.com/libre-tube/LibreTube/assets/161596117/f1ddb0e0-45c8-4501-9c31-be5cff5d2ecc

](url)

**Reply by FaqatZubair1** [reply]
Check this out, you can see the touch points that are very close to the queue button and yet it's unresponsive. You know what would help, install vimusic[https://github.com/vfsfitvnm/ViMusic/releases/download/v0.5.4/app-release.apk](url). It has better UI, and it'll help you understand what I mean. You can also add the swipe up feature, in which, if we  swipe up from the bottom bar, we can view the queue.
